lib/smile-identity-core/signature.rb

```diff
@@ -11,7 +11,8 @@ def initialize(partner_id, api_key)
     # Generates a signature based on the specified timestamp (uses the current time by default)
     #
     # @return [Hash] containing both the signature and related timestamp
-    def generate_signature(timestamp = Time.now.to_s)
+    # ISO 8601 formatted timestamp
+    def generate_signature(timestamp = Time.now.utc.strftime('%Y-%m-%dT%H:%M:%S.%LZ'))
```

Comment:
The strictest validation we have on the datetime formatting is on the biometric server. As long as it would work for our strict mode (form data) jobs it will be fine.

Reply:
Addressed